packages/react-composites/src/composites/MeetingComposite/MeetingCallControlBar.tsx

```diff
@@ -114,10 +112,58 @@ export const MeetingCallControlBar = (props: MeetingCallControlBarProps): JSX.El
               data-ui-id="meeting-composite-people-button"
               disabled={props.disableButtonsForLobbyPage}
               label={meetingStrings.peopleButtonLabel}
+              styles={commonButtonStyles}
+            />
+          )}
+          {isEnabled(meetingCallControlOptions?.chatButton) !== false && (
+            <ChatButton
+              checked={props.chatButtonChecked}
+              showLabel={true}
+              onClick={props.onChatButtonClicked}
+              data-ui-id="meeting-composite-chat-button"
+              disabled={props.disableButtonsForLobbyPage}
+              label={meetingStrings.chatButtonLabel}
+              styles={commonButtonStyles}
             />
           )}
-        </Stack.Item>
+        </Stack>
       )}
     </Stack>
   );
 };
+
+const buttonContainerStyle: IStyle = {
+  padding: '0.75rem',
+  columnGap: '0.5rem'
+};
+
+const controlBarStyles: BaseCustomStyles = {
+  root: buttonContainerStyle
+};
+
+const commonButtonStyles: ControlBarButtonStyles = {
+  root: {
+    border: 'solid 1px #E1DFDD',
```

Comment:
Why are we not using a theme palette color for this border?

Reply:
Done.